### PR TITLE
Keras is TF first. Fix TH first example

### PR DESCRIPTION
**Pull request by EderSantana**
@tboquet also, did you get this to work on TF using keras 1.1?

### CODE REVIEW

**Comment by fchollet** [comment]
Actually all examples should be made compatible with both dim orderings (they should adapt to the user's settings). So that would be the correct fix here. See other examples.

If you want to force-change the dim ordering setting within a script, it's safer to use `K.set_image_dim_ordering('th')` at the beginning than to specify a `dim_ordering` everywhere, you're less likely to miss something.

**Comment by EderSantana** [comment]
> If you want to force-change the dim ordering setting within a script, it's safer to use K.set_image_dim_ordering('th') at the beginning than to specify a dim_ordering everywhere, you're less likely to miss something.

ha! good to know! I'll do that

**Comment by EderSantana** [comment]
I user told me he was not able to use this example. This is a temporary fix. I'm investigating the tensorflow side of it now

**Comment by fchollet** [comment]
Ok as a temporary fix.

**Comment by tboquet** [comment]
@EderSantana, my apologies I extended the model architecture but the script was only working with Theano before.
 I tested it on both backend by manually changing the order of the dimensions though.
Not sure how to handle the dim ordering with parsimony in this script because of the `output_shape`.

**Comment by EderSantana** [comment]
@tboquet well, as long as it works if you manually change everything, its fine. I tried to change it all and run on TF but had no luck. Maybe I missed a layer or something

**Comment by tboquet** [comment]
@EderSantana I will verify this as soon as I have access to my laptop to be sure I did not miss something.
